server/src/analyzer/binder.ts

```diff
@@ -1072,9 +1141,9 @@ export class Binder extends ParseTreeWalker {
                         const namePartValue = node.module.nameParts[i].value;
 
                         // Is there an existing loader action for this name?
-                        let loaderActions = curLoaderActions.implicitImports ?
-                            curLoaderActions.implicitImports.get(namePartValue) :
-                            undefined;
+                        let loaderActions = curLoaderActions.implicitImports
+                            ? curLoaderActions.implicitImports.get(namePartValue)
```

Comment:
Is there a way to configure prettier to put operators at the end of the line instead of the beginning of the next line? I find this way less readable.